src/svm/interface/svm.c

```diff
@@ -184,23 +188,25 @@ PetscErrorCode SVMDestroy(SVM *svm)
 @*/
 PetscErrorCode SVMSetFromOptions(SVM svm)
 {
-  PetscErrorCode      ierr;
-
+  PetscInt            svm_mod;
+  SVMLossType         loss_type;
+  PetscInt            penalty_type;
+  PetscReal           C;
 
   PetscBool           hyperoptset;
   ModelScore          hyperopt_score_types[7];
-  PetscInt            n;
-
-  PetscInt            penalty_type;
-  PetscReal           C,logC_min,logC_max,logC_base;
-  PetscBool           flg,warm_start;
-
-  SVMLossType         loss_type;
-  PetscInt            svm_mod;
+  PetscReal           logC_base,logC_stride[3];
 
+  /* TODO change CrossValidationType to shorter CVType */
```

Comment:
We should always favour readability over brevity. So let us keep the long but descriptive name.

Reply:
Accepted.